### PR TITLE
20240625 - defines for EEPROM Addresses and EEPROM Usage Map

### PR DESCRIPTION
**Pull request by MrMep**
As per title. If this is ok, next one will be for MIDI constants.

### CODE REVIEW

File: warbl2_firmware/Functions.ino [hunk 1]
```diff
@@ -2985,11 +2987,14 @@ void saveCalibration() {
 
 // Load the stored sensor calibrations from EEPROM
 void loadCalibration() {
-    for (byte i = 0; i < 9; i++) {
-        byte high = readEEPROM((i + 9) * 2);
-        byte low = readEEPROM(((i + 9) * 2) + 1);
+    // for (byte i = 0; i < 9; i++) {
+    //     byte high = readEEPROM((i + 9) * 2);
+    //     byte low = readEEPROM(((i + 9) * 2) + 1);
+    for (byte i = EEPROM_SENSOR_CALIB_START; i < EEPROM_SENSOR_CALIB_START+18; i+2) {
```
Comment (essej) [inline warbl2_firmware/Functions.ino]: This isn't quite right i+2 should be i+=2, at least... but the whole thing may need a once-over to make sure it is equivalent too...

**Comment by amowry** [comment]
Thanks to you both. I'll take a look at this tomorrow morning. I'm not sure the best way to test it but at a minimum I'll do a visual check and then save/reload all the settings and look for any inconsistencies.

File: warbl2_firmware/Functions.ino (hunk 1 above)
Comment (MrMep) [reply warbl2_firmware/Functions.ino]: You're right, thank you!

EDIT: it should be fixed: the indexing was also wrong.

**Comment by amowry** [comment]
This looks great--thanks again!

**Pull request merged**